test/stripe.spec.ts

```diff
@@ -239,17 +265,17 @@ describe('Stripe Module', function() {
     describe('uname', () => {
```

Comment:
Should this be moved to PlatformFunctions tests?

Reply:
I don't think it should, since it tests `getClientUserAgentSeeded` which is a method on the Stripe object that's indirectly (in `getClientUserAgent`) used in `RequestSender`.

Reply:
Ok, then instead of messing with `_platformFunctions` internals we can replace it with a new implementation of PlatformFunctions that will return whatever we'd like from uname. [Something like this](https://github.com/stripe/stripe-node/blob/3c9e553ded16c8bc39f18c3264e43ffb59b38896/testUtils/index.js#L62)

Reply:
Done in https://github.com/stripe/stripe-node/pull/1660/commits/c258657a20f97081e9498a4fe568a2d6b0a33056